raft.go

```diff
@@ -937,8 +1110,8 @@ func (r *Raft) leaderLoop() {
 				v.respond(nil)
 			}
 
-		case p := <-r.peerCh:
-			p.respond(ErrLeader)
+		case future := <-r.configurationChangeChIfStable():
```

Comment:
This'll need another event to kick the loop if the channel is `nil`. Any chance this can get stuck?